spec/interactors/omniauth/google/find_or_create_user_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+require "google/apis/oauth2_v2"
+
+describe Omniauth::Google::FindOrCreateUser do
+  include_context "with interactor"
+
+  let(:initial_context) { { user_info: user_info } }
+
+  let(:user_info) do
+    instance_double "Google::Apis::Oauth2V2::Userinfo"
```

Comment:
Would you mind to add some methods for this `instance_double`? Then `before` block below could be shortened.
```suggestion
    instance_double("Google::Apis::Oauth2V2::Userinfo", given_name: "FirstName", family_name: "LastName",
      email: "user@flatstack.com", picture: "spec/fixtures/images/avatar.jpg")